serverless/admin_guide/serverless-cluster-admin-eventing.adoc

```diff
@@ -1,12 +1,12 @@
 include::modules/serverless-document-attributes.adoc[]
 [id="serverless-cluster-admin-eventing"]
-= Creating Eventing components as a cluster administrator
+= Creating Eventing components in the Administrator perspective
 include::modules/common-attributes.adoc[]
 :context: serverless-cluster-admin-eventing
 
 toc::[]
 
-If you have cluster administrator permissions on a {product-title} cluster, you can create Knative Eventing components with {ServerlessProductName} in the *Administrator* perspective of the web console or by using the `kn` and `oc` CLIs.
+You can create Knative Eventing components with {ServerlessProductName} in the *Administrator* perspective of the web console, or by using the `kn` and `oc` CLIs.
```

Comment:
You state that I can use _the `kn` and `oc` CLIs_, but you don't tell me how. Should we remove _or by using the `kn` and `oc` CLIs._ and just show the one way to create the components?

Reply:
Removed those references since this section applies just to admin actions in the Administator perspective